### PR TITLE
Update babel instructions to babel 7

### PR DESCRIPTION
**Pull request by KidkArolis**

### CODE REVIEW

**Comment by jeffvandyke** [comment]
I think you need to have `@babel/core` before the other packages in these instructions, per the [@babel/cli docs](https://babeljs.io/docs/en/babel-cli); I had to do it myself to get my project to work.

Also, good thing I checked for this PR - I was going to do a fix myself; I just got bit by these out-of-date instructions, probably could have saved me ~20 minutes figuring out why adding [@babel/node](https://babeljs.io/docs/en/babel-node) (relying on v7) was breaking everything.

File: website/en/docs/_install/compiler-babel.md [hunk 1]
```diff
@@ -2,19 +2,19 @@
...
+{{include.install_command}} @babel/cli @babel/preset-flow
 ```
 
 Next you need to create a `.babelrc` file at the root of your project with
 `"flow"` in your `"presets"`.
 
 ```json
 {
-  "presets": ["flow"]
+  "presets": ["@babel/flow"]
```
Comment (athomann) [inline website/en/docs/_install/compiler-babel.md]: I think this needs to be `@babel/preset-flow' ?

Comment (jeffvandyke) [reply website/en/docs/_install/compiler-babel.md]: According to <https://babeljs.io/docs/en/babel-preset-flow>, I would agree!

**Comment by KidkArolis** [comment]
Corrected to say @babel/preset-flow

**Comment by jeffvandyke** [comment]
Any updates? It's been 2 weeks, and these docs are still out-of-date.

**Comment by jeffvandyke** [comment]
Just in case it wasn't clear in the review feedback, I think @babel/core still needs to be installed according to <https://babeljs.io/docs/en/babel-cli>.

**Comment by KidkArolis** [comment]
Sorry, didn't realise anything was missing.

Added @babel/core to the list of pkgs to install.

Let me know if anything else is missing, I haven't used flow in a few weeks now, so not sure anymore..

**Comment by carlosrberto** [comment]
Any update here?I
I think many people are following the wrong instructions in documentation (like me)

**Comment by goodmind** [comment]
Isn't there is duplicates of this? #6952 #6396

**Comment by gkz** [comment]
Was updated form what I see

**Pull request closed without merging**